lib/ansiblelint/__init__.py

```diff
@@ -140,8 +140,10 @@ def matchyaml(self, file, text):
 
 class RulesCollection(object):
 
-    def __init__(self):
+    def __init__(self, rulesdirs=[]):
+        self._update_rulesdirs(rulesdirs)
         self.rules = []
+        self._update_rules(self.rulesdirs)
```

Comment:
`rulesdirs` is already accessible via `self` inside that method. You probably don't need to pass it as an arg. Also, if you look at the code in init, it's not obvious where this attribute appears from.